plugins/volumeFilter.go

```diff
@@ -90,6 +89,24 @@ func makeFilterVolume(
 	}, nil
 }
 
+func makeDailyVolumeByDateQuery(
```

Comment:
The extraction was to generate needed queries for test cases. This avoids repeating the same 5-6 lines, when we vary the market IDs or account IDs. I didn't add the action to sell or buy in this PR because there's no notion of "buy" in the codebase yet - it would be added in the subsequent PR that tests buy out.

With that said, I don't think this matters if we constrain the test cases, as suggested below, and reduce database calls.